packages/typespec-vscode/test/extension/common/commonSteps.ts

```diff
@@ -0,0 +1,355 @@
+      return yesdescriptionText !== null && (yesdescriptionText.includes("YesSelected folder"))
+    },
+    "Failed to match the description for the non-empty folder cases",
+    1
+  )
+  const __dirname = path.dirname(fileURLToPath(import.meta.url));
+  await page.screenshot({ path: path.resolve(__dirname, "../../images-linux/not_empty_folder_continue.png") })
+  await yesBtn!.click()
+}
+
+/**
+ * Install plugins directly from vscode
+ * @param page vscode object
+ */
+async function installExtension(page: Page) {
+  await page
+    .getByRole("tab", { name: /Extensions/ })
+    .locator("a")
+    .click()
+  await page.keyboard.type("Typespec")
+  await page
+    .getByLabel(/TypeSpec/)
+    .getByRole("button", { name: "Install" })
+    .click()
+  await page.getByRole("button", { name: "Trust Publisher & Install" }).click()
+  await sleep(10)
+  await page
+    .getByRole("tab", { name: /Explorer/ })
+    .locator("a")
+    .click()
+}
+
+/**
+ * Install plugins directly from a local file
+ * @param page vscode object
+ * @param fullFilePath The absolute address of the plugin `vsix` needs to be obtained using the path.resolve method
+ */
+async function installExtensionForFile(page: Page, fullFilePath: string) {
+  const __dirname = path.dirname(fileURLToPath(import.meta.url));
+  await page.screenshot({ path: path.resolve(__dirname, "../../images-linux/open_vscode.png") })
+  await page
+    .getByRole("tab", { name: /Extensions/ })
+    .locator("a")
+    .click()
+  await page.screenshot({ path: path.resolve(__dirname, "../../images-linux/change_extension.png") })
+  let moreItem: Locator
+  await retry(
+    page,
+    10,
+    async () => {
+      moreItem = page.getByLabel(/Views and More Actions/).first()
+      return (await moreItem.count()) > 0
+    },
+    "Failed to find more item",
+    1
+  )
+  await moreItem!.click()
+  await page.screenshot({ path: path.resolve(__dirname, "../../images-linux/more_item.png") })
+  let fromInstall: Locator
+  await retry(
+    page,
+    10,
+    async () => {
+      fromInstall = page.getByLabel(/Install from VSIX/).first()
+      return (await fromInstall.count()) > 0
+    },
+    "Failed to find install from VSIX item",
+    1
+  )
+  await fromInstall!.click()
+  await selectFolder(page, fullFilePath)
+  await retry(
+    page,
+    30,
+    async () => {
+      const installed = page.getByText(/Completed installing/).first()
+      return (await installed.count()) > 0
+    },
+    "Failed to find installed status",
+    1
+  )
+  await page.screenshot({ path: path.resolve(__dirname, "../../images-linux/extension_installed.png") })
+  await sleep(5)
+  await page
+    .getByRole("tab", { name: /Explorer/ })
+    .locator("a")
+    .click()
+  await page.screenshot({ path: path.resolve(__dirname, "../../images-linux/change_explorer.png") })
+}
+
+/**
+ * Install plugins using the command
+ */
+async function installExtensionForCommand(page: Page, extensionDir: string) {
```

Comment:
same for this and installExtensionForFile

Reply:
The function `installExtensionForFile` is no longer needed and has been removed. For function `installExtensionForCommand`, it needs to be retained as it is used to install the latest typespec extension via the code command to run UI cases.